pkg/config/services.go

```diff
@@ -915,7 +915,7 @@ var SupportedServices = serviceConfigs{
 			aws.String("events"),
 		},
 		DimensionRegexps: []*regexp.Regexp{
-			regexp.MustCompile(":rule/(?P<EventBusName>[^/]+)/(?P<RuleName>[^/]+)$"),
+			regexp.MustCompile(":rule/(?P<EventBusName>.+)/(?P<RuleName>[^/]+)$"),
```

Comment:
I'm wondering whether we need a separate regex for partner events instead, like e.g.

```
		DimensionRegexps: []*regexp.Regexp{
			regexp.MustCompile(":rule/(?P<EventBusName>[^/]+)/(?P<RuleName>[^/]+)$"),
			regexp.MustCompile(":rule/aws.partner/(?P<EventBusName>.+)/(?P<RuleName>[^/]+)$"),
		},
```

Can you check if this works for you?

Reply:
Thanks @cristiangreco, according to the docs it seems like partner busses are always prepended with aws.partner: https://docs.aws.amazon.com/eventbridge/latest/onboarding/amazon_eventbridge_partner_onboarding_guide.html#AWS_EventBridge_Partner_Onboarding_Names
I'll update the PR as suggested.